reduct/boolean-reduct/rte-helpers.metta

```diff
@@ -93,6 +93,31 @@
 )
 
 (= (getGsetAndChildren $expr) (GsetandChildren $expr () ()))
```

Comment:
I don't think this line here is your function. Can you remove it if it's not needed? It may cause issues in the future